source/api/presentation/3.0/index.md

```diff
@@ -1738,7 +1738,7 @@ While any HTTP(S) URI is technically acceptable for any of the resources in the
 
 ###  6.2. Requests
 
-Clients _MUST NOT_ attempt to construct resource URIs by themselves, instead they _MUST_ follow links from within retrieved descriptions or elsewhere.
+Clients _MUST NOT_ attempt to construct URIs for Presentation API resources by themselves, instead they _MUST_ follow links from within retrieved descriptions or elsewhere. Other services _MAY_ require constructing URIs according to their specifications.
 
```

Comment:
Do you want to call out the Image API specifically, something like:
>Other IIIF services (e.g. the [Image API](https://iiif.io/api/image)) _MAY_ require ...

Reply:
How about something less normative:

> Clients are expected to follow links to Presentation API resources from within retrieved descriptions or elsewhere. Unlike Image API requests, the URIs for Presentation API resources cannot be assumed to follow any particular pattern.

I don't really like using untestable normative language about clients any more than we have to.